packages/psss/src/containers/Modals/ConfirmModal.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import ReportProblem from '@material-ui/icons/ReportProblem';
+import Typography from '@material-ui/core/Typography';
+import {
+  Button,
+  OutlinedButton,
+  Dialog,
+  DialogFooter,
+  DialogHeader,
+  DialogContent,
+  LoadingContainer,
+  Alert,
+} from '@tupaia/ui-components';
+
+const DescriptionText = styled(Typography)`
+  font-size: 1rem;
+  margin-top: 1rem;
+`;
+
+const ConfirmIcon = styled(ReportProblem)`
+  font-size: 2.5rem;
+  margin-bottom: 0.3rem;
+  color: ${props => props.theme.palette.primary.main};
+`;
+
+const PositionedAlert = styled(Alert)`
+  margin-top: 1rem;
+`;
+
+export const ConfirmModal = ({
```

Comment:
This is a great, generic component!! It could potentially be used in place of similar components (such as `AlertCreateModal`), mainly for styling/structural consistency. No need to do so now, of course